client/extensions/zoninator/state/data-layer/zones/index.js

```diff
@@ -13,6 +13,7 @@ import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
 import { errorNotice, removeNotice, successNotice } from 'state/notices/actions';
 import { getSiteSlug } from 'state/sites/selectors';
 import { requestError, updateZone, updateZones } from '../../zones/actions';
+import { parseZoneResponse, parseZonesResponse } from './util';
```

Comment:
A small thing, but I usually see these files named `utils.js` rather than `util.js`.

Also, in the data layer they've standardized on using `fromApi` for fetching and `toApi` for saving. However, I get why you've used the naming convention you did. I wonder though if, in order to better adhere to the established naming standards, `parseZoneResponse` could be renamed to `fromApi`, and then `parseZoneReponses` could be removed altogether by rolling its functionality into the `dispatch` call. Something like:

`dispatch( updateZones( siteId, map( data, fromApi ) ) );`